app/gen-server/migration/1756918816559-ServiceAccounts.ts

```diff
@@ -0,0 +1,97 @@
+import { MigrationInterface, QueryRunner, Table, TableColumn, TableForeignKey, TableIndex } from "typeorm";
+import * as sqlUtils from "app/gen-server/sqlUtils";
+
+export class ServiceAccounts1756918816559 implements MigrationInterface {
+
+  public async up(queryRunner: QueryRunner): Promise<any> {
+    const dbType = queryRunner.connection.driver.options.type;
+    const datetime = sqlUtils.datetime(dbType);
+
+    await queryRunner.addColumns('users', [
+      new TableColumn({
+        name: 'type',
+        type: 'varchar',
```

Comment:
[One existing migration](https://github.com/gristlabs/grist-core/blob/445fab23b251f884f73b0df766d3ce26d8065f10/app/gen-server/migration/1753088213255-GroupTypes.ts#L10) used an `enum:` specification for a `type` column. This does not seem to have any effect on the resulting schema (at least for PostgreSQL) but seems helpful to document the allowed values?